api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -67,19 +76,41 @@ public void saveAlert_shouldSaveAlertToDb() {
 	@Test
 	public void getAlert_shouldGetAlertById() {
 		Alert savedAlert = hibernateAlertDAO.getAlert(2);
+		Date currentDate = new Date();
+		dateFormat.format(currentDate);
+		Calendar c = Calendar.getInstance();
+		c.setTime(currentDate);
+		c.add(Calendar.YEAR,1);
+		Date currentDatePlusOneYear = c.getTime();
+		savedAlert.setDateToExpire(currentDatePlusOneYear);
```

Comment:
I think it's highly preferable to use the `java.time` classes here. There's really no reason to keep using `Calendar` and `Date`, e.g.,

```java
savedAlert.setDateToExpire(Date.from(ZonedDateTime.now().plusYears(1).toInstant()));
```